src/Serilog/Context/LogContext.cs

```diff
@@ -12,9 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 #if LOGCONTEXT
```

Comment:
This a site @khellang pointed me to.  Not always 100% however a great start.

- http://packagesearch.azurewebsites.net/?q=CallContext
- http://packagesearch.azurewebsites.net/?q=AsyncLocal

These would indicate `5.1` could be difficult.